docs/features/user-watchlists.md

```diff
@@ -69,8 +69,13 @@ To ensure a **cross-device experience**, we standardize on the following datafor
 
 ## Watchlist playlist
 
+<<<<<<< HEAD
```

Comment:
It looks like the merge has some merge conflict artifacts

Reply:
Fixed it!

Reply:
I still see 2 instances of "The media metadata..." line as well as a line that says:

> > > > > > > > f949d534c5d8a7e25032b20b4ff74f74ec6ce453

Reply:
:-( I guess I'm missing some understanding of the Git(hub) workflow here.